gossip/minimal/goshawk.go

```diff
@@ -535,6 +543,9 @@ func (o *originLog) validateSTH(ctx context.Context, sthInfo *x509ext.LogSTHInfo
 		TreeHeadSignature: sthInfo.TreeHeadSignature,
 	}
 	if err := o.Log.VerifySTHSignature(sth); err != nil {
+		o.reporter.Logf(ctx, o.URL, "STH signature verification failure", "signature",
+			fmt.Sprintf("%s/ct/v1/get-sth", o.URL),
```

Comment:
nit: Could use the constants in types.go: https://github.com/google/certificate-transparency-go/blob/97c991c7a5f737f3199bec2af29cacf4e48a898b/types.go#L434-L445